src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/index.styles.ts

```diff
@@ -304,6 +311,9 @@ export const useStyles = makeStyles((theme: Theme) => ({
             return `1px solid ${theme.colors.tint90}`;
         },
         color: (props: any) => {
+            if (props.step === 5) {
```

Comment:
Do not use constants, create enum with steps and use them in all the places

Reply:
done